src/components/flow/actions/callbrain/CallBrainForm.tsx

```diff
@@ -14,15 +15,36 @@ import { BrainInfo } from '../../../../store/flowContext';
 import { applyVueInReact } from 'veaury';
 // @ts-ignore
 import Unnnic from '@weni/unnnic-system';
+import TextInputElement, {
+  TextInputSizes,
+} from 'components/form/textinput/TextInputElement';
+import { StringEntry } from 'store/nodeEditor';
 const UnnnicIcon = applyVueInReact(Unnnic.unnnicIcon);
 
 export interface CallBrainFormProps extends ActionFormProps {
   brainInfo: BrainInfo;
+  entry: string;
 }
 
-export class BrainForm extends React.Component<CallBrainFormProps> {
+export interface CallBrainFormState {
+  entry: StringEntry;
+}
+
+export class BrainForm extends React.Component<
+  CallBrainFormProps,
+  CallBrainFormState
+> {
+  constructor(props: CallBrainFormProps) {
+    super(props);
+    this.state = { entry: { value: this.props.entry || '' } };
```

Comment:
You should follow the floweditor pattern to handle node to state data

Use other actions as example, such as [AddLabels](https://github.com/weni-ai/floweditor/blob/main/src/components/flow/actions/addlabels/AddLabelsForm.tsx#L34)
With this the `mapStateToProps` modification below should be removed and the `entry prop` above too